xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Translations.xml

```diff
@@ -176,9 +182,46 @@ rendering.macro.taglist.notags=As you and your team add tags to the content, thi
 #### Confluence Bridges in the generic pro macro package
 
 ## Content Report Table macro
+rendering.macro.content-report-table.name=Content Report Table
+rendering.macro.content-report-table.description=Provides a content report in table format, based on labels.
+rendering.macro.content-report-table.parameter.labels.name=Label
+rendering.macro.content-report-table.parameter.labels.description=Label(s) to report on
```

Comment:
I think we could change the translations to use the concept of "tag" instead of "label".